### DISCUSSION TITLE
aarch64 should be in the free edition, its an industry standard now

### DISCUSSION THREAD

**Discussion by MCJEModder2026** [Vector35/binaryninja-api#5934] [Ideas] [answered]
i dont want to pay $300 for an app i need to use once in a blue moon

**Comment by psifertex** [comment, marked as answer]
There's always [cloud](https://cloud.binary.ninja/) for that. 

If we include all standard architectures there's far fewer reasons why anyone would buy the product and we would have a hard time funding any improvements.

At some point we could swap out aarch64 for one of the other free architectures but that would likely upset a different subset of folks. That said, which arch would you recommend we swap it for if we were to do that?

**Comment by MCJEModder2026** [comment]
armv7, theese days the main arch are x86_64 and apple silicon/aarch64 
this may upset people with old android phones/tablets or old rasberry pis

**Comment by MCJEModder2026** [comment]
> There's always [cloud](https://cloud.binary.ninja/) for that. 
> 
> 
> 
> If we include all standard architectures there's far fewer reasons why anyone would buy the product and we would have a hard time funding any improvements.
> 
> 
> 
> At some point we could swap out aarch64 for one of the other free architectures but that would likely upset a different subset of folks. That said, which arch would you recommend we swap it for if we were to do that?

Swap it with Arm32, most modern devices are arm64 now 
Binja cloud has a file size limit

**Comment by iSavra** [comment]
@psifertex I totally get where you're coming from, but can Vector35 reconsider this decision? 
Windows on Arm will become standard now that NVIDIA and Qualcomm are releasing aarch64 SoCs, and Apple just dropped support for X86 devices, I'd say armv7 is less important nowadays, you could swap those architectures if you wanna give people reasons to upgrade.

Thanks in advance!
